collect_app/src/main/java/org/odk/collect/android/utilities/FileUtils.java

```diff
@@ -180,69 +167,62 @@ public static String getMd5Hash(InputStream is) {
         }
     }
 
-
-    public static Bitmap getBitmapScaledToDisplay(File f, int screenHeight, int screenWidth) {
-        // Determine image size of f
-        BitmapFactory.Options o = new BitmapFactory.Options();
-        o.inJustDecodeBounds = true;
-        getBitmap(f.getAbsolutePath(), o);
-
-        int heightScale = o.outHeight / screenHeight;
-        int widthScale = o.outWidth / screenWidth;
-
-        // Powers of 2 work faster, sometimes, according to the doc.
-        // We're just doing closest size that still fills the screen.
-        int scale = Math.max(widthScale, heightScale);
-
-        // get bitmap with scale ( < 1 is the same as 1)
-        BitmapFactory.Options options = new BitmapFactory.Options();
-        options.inInputShareable = true;
-        options.inPurgeable = true;
-        options.inSampleSize = scale;
-        Bitmap b = getBitmap(f.getAbsolutePath(), options);
-        if (b != null) {
-            Timber.i("Screen is %dx%d.  Image has been scaled down by %d to %dx%d",
-                    screenHeight, screenWidth, scale, b.getHeight(), b.getWidth());
-        }
-        return b;
-    }
-
     /**
-     * With this method we do not care about efficient decoding and focus
-     * more on a precise scaling to maximize use of space on the screen
+     * Scales image according to the given display
+     *
+     * @param file            containing the image
+     * @param screenHeight    height of the display
+     * @param screenWidth     width of the display
+     * @param scaleAccurately determines whether to scale more precisely without caring about
+     *                        efficient decoding and to maximize use of space on the screen
+     * @return scaled bitmap
      */
-    public static Bitmap getBitmapAccuratelyScaledToDisplay(File f, int screenHeight,
-                                                            int screenWidth) {
-        // Determine image size of f
-        BitmapFactory.Options o = new BitmapFactory.Options();
-        o.inJustDecodeBounds = true;
-        getBitmap(f.getAbsolutePath(), o);
-
-        // Load full size bitmap image
+    public static Bitmap getBitmapScaledToDisplay(File file, int screenHeight, int screenWidth, boolean scaleAccurately) {
+        // Determine image size of file
         BitmapFactory.Options options = new BitmapFactory.Options();
-        options.inInputShareable = true;
-        options.inPurgeable = true;
-        Bitmap bitmap = getBitmap(f.getAbsolutePath(), options);
+        options.inJustDecodeBounds = true;
+        getBitmap(file.getAbsolutePath(), options);
 
-        // Figure out scale
-        double heightScale = ((double) (o.outHeight)) / screenHeight;
-        double widthScale = ((double) o.outWidth) / screenWidth;
-        double scale = Math.max(widthScale, heightScale);
+        Bitmap bitmap;
+        double scale;
```

Comment:
I recommend this change, so you can make bitmap and scale blank finals:

![screen shot 2018-03-18 at 7 46 40 pm](https://user-images.githubusercontent.com/55099/37575641-6fbb41a8-2ae5-11e8-853b-5c47276d43ac.png)